girder/__init__.py

```diff
@@ -262,6 +263,29 @@ def logprint(*args, **kwargs):
         _originalStdOut.flush()
 
 
+def _setupCache():
```

Comment:
Is this function idempotent? It would be great if we could guarantee that it was, since it's called during server setup. See #2530

Reply:
The global variables (cache and requestCache) are simple interfaces to the cache backends, the backends are completely replaced on each server startup.